modules/configure_repo/main.ruleset_main.tf

```diff
@@ -0,0 +1,27 @@
+resource "github_repository_ruleset" "main" {
+  name        = "main"
+  repository  = github_repository.this.name
+  target      = "branch"
+  enforcement = "active"
+
+  conditions {
+    ref_name {
+      include = ["~DEFAULT_BRANCH", "refs/heads/main"]
+      exclude = []
+    }
+  }
+
```

Comment:
Please can you add `bypass_actors` block for the avm tf terraform team?

Reply:
We'll need to update the spec if we do.  I believe it currently states that no-one is allowed to bypass.